### PR TITLE
Fixed using proper proxy auth credentials when updating the database and talking to the WPScan API.

### PR DESCRIPTION
**Pull request by ArSn**
The TL;DR of this is that this was calling [a wrong function](https://github.com/wpscanteam/CMSScanner/blob/master/lib/cms_scanner/browser.rb#L42) where the `proxyuserpwd` parameter [does not get added](https://github.com/wpscanteam/CMSScanner/blob/master/lib/cms_scanner/browser.rb#L79) in the CMSScanner gem.

## Licensing

By submitting code contributions to the WPScan development team via Github Pull Requests, or any other method, it is understood that the contributor is offering the WPScan company (company number 	83421476900012), which is registered in France, the unlimited, non-exclusive right to reuse, modify, and relicense the code.

### CODE REVIEW

**Comment by jwidavid** [comment]
Love simple fixes!

**Review by jwidavid — APPROVED** [review]
